api/src/routes/public/email-subscription.ts

```diff
@@ -61,7 +61,9 @@ export const emailSubscribtionRoutes: FastifyPluginCallbackTypebox = (
         );
 
         await Promise.all(userUpdatePromises);
-        log.info('Successfully unsubscribed users from email');
+        log.info(
+          `Successfully unsubscribed users from email: ${unsubUsers.map(user => user.email).join(', ')}`
+        );
```

Comment:
This is a nit, but I think majority of the logs look nicer is the dynamic data is logged as such:

```suggestion
        log.info({ emails: unsubUsers.map(u => u.email) }, 'Successfully unsubscribed users from email.');
```

I would not nit the same about `Found ${unsubUsers.length} users ...`, because that is simple/small enough to not make enough of a difference.

_Again, up to you_

Reply:
Fair enough.